ui/src/app/cohort-search/cohort-builder/cohort-builder.component.ts

```diff
@@ -26,17 +27,18 @@ import {
 })
 export class CohortBuilderComponent implements OnInit, OnDestroy {
 
-  @select(includeGroups) includeGroups$;
-  @select(excludeGroups) excludeGroups$;
-  @select(wizardOpen) open$;
-  @select(totalCount) total$;
-  @select(isRequstingTotal) isRequesting$;
+  @select(includeGroups) includeGroups$: Observable<List<any>>;
+  @select(excludeGroups) excludeGroups$: Observable<List<any>>;
```

Comment:
Should these list be of type SearchGroup? instead of any?

Reply:
No; they are not lists of `SearchGroup`s.  They are `List`s of immutable `Map`s that contain more information than a `SearchGroup`.  I have not yet created typings for the full store beyond the immutable types. These typings are included to make explicit that the top level API for the data structure is `List` and not, say, `Map` or `Set`.